Sources/tart/Commands/Get.swift

```diff
@@ -0,0 +1,31 @@
+import ArgumentParser
+import Foundation
+
+struct Get: AsyncParsableCommand {
+  static var configuration = CommandConfiguration(commandName: "get", abstract: "Get a VM's configuration")
+
+  @Argument(help: "VM name")
+  var name: String
+
+  func run() async throws {
+    do {
+      let vmDir = try VMStorageLocal().open(name)
+      let vmConfig = try VMConfig(fromURL: vmDir.configURL)
```

Comment:
what happens if I pass a `name` that doesn't exist `tart get sepllingmistake`

Reply:
would that bubble up via the `throws` keyword? Apologies if my Swift knowledge isn't up to scratch

Reply:
It does bubble up but the error message is somewhat cryptic. 

```
> ./tart get masdf
UninitializedVMDirectoryError()
```

However, this is in line with what happens from an incorrect name using `tart set`, which is why I did not modify it